mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 # Project information
-site_name: Arista AVD collection
+site_name: Arista AVD
 site_author: Arista Ansible Team
 site_description: Arista Validated Design collection's documentation
```

Comment:
Still says collection

Reply:
You resolved this one with no update. I have comitted a change to line 4 to remove the "collection's" and add "s" on "Designs". I will approve now, but let me know if there was a reason why you didn't change the description.